pip_connectors/Makefile

```diff
@@ -0,0 +1,20 @@
+V = 0
+Q = $(if $(filter 1,$V),,@)
+
+ifeq ($(OS),Windows_NT)
+  VENV_PATH_DIR = Scripts
+else
+  VENV_PATH_DIR = bin
+endif
+
+pip-vesselmovement-run:
+	$(Q) docker-compose -f docker/docker-compose.yaml up --build vesselmovement
```

Comment:
Latest docker compose command is actually a subcommand of docker -> `docker compose`
I would separate out the `docker-compose` command to a variable so its easy to change if needs be

Reply:
latest is installed on the ec2 machine